package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dhis2-home-page",
```

Comment:
Why are you keeping "dhis2" in the name of the app? The zip file generated during the build process is named "dhis2-home-page" which may be confusing for the user